### PR TITLE
Style compare grid

### PR DESCRIPTION
**Pull request by jakelauer27**
compare.js - Refactored SortDown function into a couple lines.

Added sticky header and arrows to compare grid. Added animations for said arrows.
Refactored the logic for adding and removing classes (up / highlight /rotate-arrow) so that they are handled by state.

### CODE REVIEW

File: src/Compare.js [hunk 1]
```diff
@@ -1,135 +1,97 @@
...
-      sortedData = this.sortUp(e, sortKey, sortKey2)
+      sortedData = this.sortDown(e);
     } else {
-      sortedData = this.sortDown(e, sortKey, sortKey2)
+      sortedData = this.sortUp(e, sortKey, sortKey2);
     }
     this.setState({
-      data: sortedData
+      data: sortedData,
+      highlight: e.target.classList[0],
```
Comment (JustinTorrance) [inline src/Compare.js]: The highlighting is a great touch.

File: src/Compare.js [hunk 2]
```diff
@@ -1,135 +1,97 @@
...
+        if(a[sortKey][sortKey2] < b[sortKey][sortKey2]) {return 1;}
         return 0;
       }
-      if(a[sortKey] > b[sortKey]) {
-        return -1;
-      }
-      if(a[sortKey] < b[sortKey]) {
-        return 1;
-      }
+      if(a[sortKey] > b[sortKey]) {return -1;}
```
Comment (JustinTorrance) [inline src/Compare.js]: Good job drying up this function. I have no suggested edits.

File: src/CompareItem.js [hunk 3]
```diff
@@ -1,30 +1,34 @@
...
-        <td className={props.row}>{props.character.speeds.run_speed}</td>
-        <td className={props.row}>{props.character.speeds.air_speed}</td>
-        <td className={props.row}>{props.character.speeds.fall_speed}</td>
-        <td className={props.row}>{props.character.speeds.fast_fall_speed}</td>
-        <td className={props.row}>{props.counter}</td>
-        <td className={props.row}>{props.character.strongest_smash.attack}</td>
-        <td className={props.row}>{props.character.strongest_smash.damage}</td>
+        <td className={this.props.row}><img src={this.props.character.images.small_icon}/></td>
+        {
+          categories.map( category => {
```
Comment (JustinTorrance) [inline src/CompareItem.js]: I have no suggested edits. Nice work!

File: src/Compare.js [hunk 4]
```diff
@@ -1,135 +1,97 @@
 import React, { Component } from 'react';
-import characters from './data';
+import { characters, categories } from './data';
```
Comment (libbyeh) [inline src/Compare.js]: Discussed addition to dataset-need to do a PR to Brittany.

File: src/Landing.js [hunk 5]
```diff
@@ -18,8 +18,10 @@ class Landing extends Component {
     if (this.state.showLanding) {
       return (
         <div className='landing-page'>
+          <div className='action-container'>
```
Comment (libbyeh) [inline src/Landing.js]: Maybe consider a section instead of multiple divs to be more semantic.

File: src/styles/App.css [hunk 6]
```diff
@@ -41,17 +9,38 @@ table, th, tr, td {
 }
 
 .compare-header {
-  color: black;
+  color: white;
```
Comment (libbyeh) [inline src/styles/App.css]: Reviewed and love the edits you made to the grid, much 'softer'. No edits

File: src/styles/_landing.scss [hunk 7]
```diff
@@ -6,9 +8,28 @@
...
+    border: 0px;
+    background: none;
+    font-family: 'Smash Bros', serrif;
+    font-size: 2em;
+    margin: 20px calc(50vw - 100px);
+    width: 200px;
+
+    &:hover {
+      color: white;
+    }
```
Comment (libbyeh) [inline src/styles/_landing.scss]: Im happy with all the styling changes. no edits

**Pull request merged**